docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.8"
+
+services:
+  todo-app:
```

Comment:
There is no bind mount to the project folder. This prevents hot-reloading from working and prevents this from working as a fully effective development container. This is also a blocker for this passing review. Please see page 6 from the exercise brief for some clues on how to set this up.